src/software/ai/hl/stp/tactic/transition_conditions.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "software/world/world.h"
+
+// Transition conditions are conditions to check before transitioning between different
+// gameplay behaviour
+
+/**
+ * Checks whether the robot is done moving
+ *
+ * @param robot The robot to check
+ * @param destination The destination to move to
+ * @param final_orientation The final orientation of the movement
+ * @param DISTANCE_THRESHOLD distance from destination to be considered close enough
+ * @param ANGLE_THRESHOLD angle from final orientation to be considered close enough
+ *
+ * @return true if robot movement is done
+ */
+bool moveRobotDone(const Robot& robot, const Point& destination,
```

Comment:
nit: this function name doesn't make it clear what the function does, maybe something like `robotReachedDestination` would be clearer

Reply:
done